spec/requests/api/moves_controller_create_v2_spec.rb

```diff
@@ -35,7 +35,8 @@
     end
 
     let(:profile) { create(:profile) }
-    let(:from_location) { create :location, suppliers: [supplier] }
+    let(:another_supplier) { create(:supplier) }
```

Comment:
You actually don't need any changes to this file :).

Reply:
we discussed this, and it makes sense to keep it